### ISSUE TITLE
facial_keypoints is predicting on training set

### ISSUE THREAD

**Issue by yasiemir** [najeeb97khan/Kaggle#1]
the code in facial ipynb is predicting on trainng set instead of test set.

**Comment by ghost** [comment]
Can u check and make it correct? But I seem to have taken another handle(df_test) for the test set.
